### ISSUE TITLE
W017 false positive

### ISSUE THREAD

**Issue by geoffroy-noel-ddh** [Riverside-Healthcare/djLint#11]
test.html

```python
<!DOCTYPE html>
<html lang="en">
    <head>
        <title>title</title>
        <meta charset="utf-8">
        <meta name="description" content="test page" />
    </head>
    <body>
        <br>
        <br/>
        <img alt="img1" src="/test1.png">
        <img alt="img2" src="/test2.png"/>
    </body>
</html>
```

```bash
djlint test.html

Linting 1/1 files 
test.html
 W017 5:8 Tag should be self closing. <meta charset="utf-8
 W006 11:8 Img tag should have alt, height and width attributes. <img alt="img1" src=
 W017 11:8 Tag should be self closing. <img alt="img1" src=

Linted 1 file, found 3 errors.
```

In HTML5 empty elements like meta and img do not need to end with />. The above html document passes W3C html validator without any warning. 

Expected output: 1 warning.

I guess I could disable the W017 rule. But wouldn't it make sense to be disabled by default? Another thing I don't understand is why djlint reports a warning for img and not br?

djlint 0.3.4

**Comment by christopherpickering** [comment]
Thanks for the catch on `<br>`, The next release will have updated regex to catch that. 

The rules are pretty inclusive and can be ignored if not needed. Many others are also not "required" by html5, line length for example.
